schemas/common_definitions.json

```
@@ -271,6 +271,11 @@
               "type": "string",
               "description": "The id of another group. The first block in that group will be routed to."
             },
+            "section": {
+              "type": "string",
+              "description": "Routes to the end of a section",
```

```suggestion
              "description": "Routes to the end of the current section",
```

Does the suggestion make more sense? The only value allowed is 'end' so I am assuming it is routing within the same section.